test/src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -1690,7 +1691,11 @@ void resolveDependencies() throws Exception {
 
                             File dst = new File(home, "plugins/" + artifactId + ".jpi");
                             if(!dst.exists() || dst.lastModified()!=dependencyJar.lastModified()) {
-                                FileUtils.copyFile(dependencyJar, dst);
+                                try {
+                                    FileUtils.copyFile(dependencyJar, dst);
+                                } catch (ClosedByInterruptException x) {
+                                    throw new AssumptionViolatedException("copying dependencies was interrupted", x);
```

Comment:
> Why not `IOException`?

Because `ClosedByInterruptException` is specifically what is observed to be thrown, and I do not want to cast too wide a net. If there are other unrelated exceptions, they should be investigated separately, not suppressed.

> Perhaps it is worth to determine the root cause

Please, be my guest. After over a year I still have no idea what is causing the test thread to be interrupted, no idea how to reproduce it locally, and no idea [how to add logging to diagnose it](http://stackoverflow.com/a/2127397/12916). In the meantime I want PRs to stop being marked as erroneous due to a random infrastructure failure.